### ISSUE TITLE
Make Re-order in Realtime Effects Actionable via Accessibility

### ISSUE THREAD

**Issue by chinakov** [audacity/audacity#8767]
### Description

If the User has engaged the focus box of real time effects and real time master effects as described in https://github.com/audacity/audacity/issues/8766, the first thing to be selected within the focus should be the re-order button. 

Once the User presses 'Enter' to focus on the re-order button, there should be a yellow border akin to the 'trim' and 'stretch' border when using clips view via accessibility, so that the User can adjust the position of the effect within the focus box. 

![Image](https://github.com/user-attachments/assets/22a61679-045b-4253-9955-edfbfbdefdeb)

**Comment by saintmatthieu** [comment]
Looks like part of https://github.com/audacity/audacity/issues/7836
@chinakov can you remember why a specific ticket was created for this?
